lib/compat/wordpress-6.3/html-api/class-wp-html-attribute-token.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
```

Comment:
Coding standard is going to need to be fixed.

Reply:
cc: @SergeyBiryukov - looks like someone missed this when approving your change.

@spacedmonkey this is code that has already been accepted in Core. can you fix the issue in Core and create a follow-up back-port into Gutenberg?

until the linting tools are fixed this is going to continue to be a recurring problem. let's not introduce breakage between the two halves of Core because our coding standards are incomplete.

Reply:
nevermind - I realized this is actually just a temporary thing introducing these new files. thanks @spacedmonkey - not going to be a problem